spec/factories/user_factory.rb

```diff
@@ -40,6 +40,27 @@
     end
   end
 
+  factory :prescription_user, class: 'User' do
```

Comment:
yea, we can tweak anything in user_factory.rb but this is satisfactory for specs I think.